src/steamship/data/file.py

```diff
@@ -209,7 +209,7 @@ def tag(
         self,
         plugin_instance: str = None,
         wait_on_tasks: List[Task] = None,
-    ) -> Task[Tag]:
+    ) -> Task[Tag]:  # This actually is Task[TagResponse], right?
```

Comment:
Nice catch -- yes!

Reply:
@EniasCailliau It doesn't look like I can just swap it to TagResponse because of the circular dependencies.  Any suggestions?